RecoHGCal/TICL/plugins/SuperclusteringSampleDumper.cc

```diff
@@ -151,7 +151,8 @@ void SuperclusteringSampleDumper::analyze(const edm::Event& evt, const edm::Even
   std::iota(trackstersIndicesPt.begin(), trackstersIndicesPt.end(), 0);
   std::stable_sort(
       trackstersIndicesPt.begin(), trackstersIndicesPt.end(), [&inputTracksters](unsigned int i1, unsigned int i2) {
-        return (*inputTracksters)[i1].raw_pt() > (*inputTracksters)[i2].raw_pt();
+        return (*inputTracksters)[i1].raw_energy() * std::sin((*inputTracksters)[i1].barycenter().Theta()) >
```

Comment:
General comment for computing the pT.  #48184 fixes the calculation of the pT, using the barycenter position, can we just revert these changes and go back to use `trackster.raw_pt()` ?